### PR TITLE
Adding virtual networks models

### PR DESCRIPTION
**Pull request by mlunoe**
This is adding the models for the Virtual Networks

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @aldipower

File: src/js/constants/ActionTypes.js [hunk 1]
```diff
@@ -1,5 +1,21 @@
 let ActionTypes = {};
 [
+  'REQUEST_CHRONOS_JOB_CREATE_ERROR',
```
Comment (jfurrow) [inline src/js/constants/ActionTypes.js]: This is just :abcd: right?

File: src/js/events/VirtualNetworksActions.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+const VirtualNetworksActions = {
+  fetch: function () {
+    RequestUtil.json({
+      url: `${Config.rootUrl}${Config.virtualNetworksApi}/state`,
+      success: function (response) {
+        let {overlays, vtep_mac_oui, vtep_subnet} = response.network;
+        // Map structure to mimic agents overlays
+        overlays = overlays.map(function (overlay) {
+          return {info: overlay};
+        });
```
Comment (jfurrow) [inline src/js/events/VirtualNetworksActions.js]: Are we sure we want to alter the data here? I'm not opposed to it, just want to make sure we are conscious about it.

File: src/js/stores/__tests__/VirtualNetworksStore-test.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+jest.dontMock('../../events/VirtualNetworksActions');
+
+import PluginTestUtils from 'PluginTestUtils';
+
+const ActionTypes = require('../../constants/ActionTypes');
+const AppDispatcher = require('../../events/AppDispatcher');
+const EventTypes = require('../../constants/EventTypes');
+const VirtualNetworksStore = require('../VirtualNetworksStore');
+const Overlay = require('../../structs/Overlay');
+const OverlayList = require('../../structs/OverlayList');
```
Comment (jfurrow) [inline src/js/stores/__tests__/VirtualNetworksStore-test.js]: :abcd:

**Comment by jfurrow** [comment]
Very minor comments. LGTM! :ship:

**Comment by rcorral** [comment]
LGTM too, 🚢  after comments are addressed.

File: src/js/constants/ActionTypes.js (hunk 1 above)
Comment (mlunoe) [inline src/js/constants/ActionTypes.js]: Yes!

File: src/js/events/VirtualNetworksActions.js (hunk 2 above)
Comment (mlunoe) [inline src/js/events/VirtualNetworksActions.js]: Yes, that is the reducer pattern that we introduced with ChronosActions: https://github.com/dcos/dcos-ui/blob/master/src/js/events/ChronosActions.js#L58

File: src/js/stores/__tests__/VirtualNetworksStore-test.js (hunk 3 above)
Comment (mlunoe) [inline src/js/stores/__tests__/VirtualNetworksStore-test.js]: done

**Pull request merged**